src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1022,6 +1022,17 @@ public void deleteStudent(String courseId, String studentEmail) {
         studentsLogic.deleteStudentCascade(courseId, studentEmail);
     }
 
+    /**
+     * Deletes all the students in the course.
+     *
+     * @param courseId course id for the students
+     */
+    public void deleteAllStudentsInCourse(String courseId) {
+
+        Assumption.assertNotNull(courseId);
+        studentsLogic.deleteAllStudentsInCourse(courseId);
```

Comment:
@whipermr5 In future (when we upgrade the student data UI) we are going to need a method that can delete a large number of students (but not necessarily all students in the course) in a scalable way. May be tasks queues is the way to go?